app/views/controllers/checklists/show.html.erb

```diff
@@ -19,7 +20,12 @@ names = @data.taxa
 <%= panel_block(inner_class: "checklist") do %>
   <ul class="list-unstyled">
     <% names.each do |name| %>
-      <li><i><%= link_to(name[0], name_path(name[1])) %></i></li>
+      <li>
+        <i><%= link_to(name[0], name_path(name[1])) %></i>
```

Comment:
Would it be more useful to link to an index of the Project's Observations of the Name?

Reply:
It would be, but I'm not sure we have an existing URL that filters observations based on project membership and name.  If there is one, I'd be happy to use it.

Reply:
Hmm, maybe i can supply from PatternSearch.
I'd also suggest using the `<i>` tag inside the link, so this:
```ruby
<%= tag.li do 
      link_to(observations_path(pattern: "project:#{@project.id} name:#{name.id") do 
        tag.i(name[0]
      end
    end %>
```
I'm pretty sure that will get you where you need to go.

Reply:
Aside: It might be cool in the future to refactor this param (or do an alternate) so we could pass stuff like 
```observations_path(filter: { project: @project.id, name: name.id })```